ratatool-sampling/src/main/scala/com/spotify/ratatool/io/BigQueryIO.scala

```diff
@@ -84,15 +84,15 @@ object BigQueryIO {
 
 }
 
-private class TableRowIterator(private val iterator: PatchedBigQueryTableRowIterator)
+private class TableRowIterator(private val iter: PatchedBigQueryTableRowIterator)
```

Comment:
curious why the rename?

Reply:
yeah this is a bit weird.  2.13 collections update added a final field called iterator to Iterators: https://github.com/scala/scala/blob/v2.13.2/src/library/scala/collection/Iterator.scala#L96

So this is now trying to override that and can't.  This maintains compatibility for both 2.12 and 2.13